qplist.pro

```diff
@@ -0,0 +1,14 @@
+QT += xml
```

Comment:
This file is misnamed according to your qt-pods requirements.  This repository is called qtplist, and the .pro file is called qplist.  I do prefer the name qtplist for the project, as it matches Qt's naming convention for modules better.